test-data/unit/check-flags.test

```diff
@@ -515,3 +515,95 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
+[case testDisallowAnyDecoratedUnannotatedDecorator]
```

Comment:
Huh, both are really cool test ideas, actually, especially the second one.

[Done](https://github.com/python/mypy/pull/3536/commits/3b7a079d189cf82da3318d20b99eb8d7e0a68b7e)!